attending/index.py

```diff
@@ -0,0 +1,19 @@
+import csv
+from pathlib import Path
+import os
+
+
+def load_index(file: Path):
+    project_index = {}
+    assert file.is_file()
+    with open(str(file), 'r') as index:
```

Comment:
no need for `str` here, and 'r' is the default mode anyway.
```suggestion
    with open(file) as urls:
```